BannerlordTwitch/BannerlordTwitch/_Module/ModuleData/Languages/DE/strings-powers-DE.xml

```diff
@@ -137,24 +140,24 @@
         <!-- Shatter Shield II -->
         <string id="Dvjkoxi3" text="Schild zerschmettern II" original="Shatter Shield II"/>
         <!-- Shatter Shield III -->
-        <string id="V6RyPIQg" text="Splitterschild III" original="Shatter Shield III"/>
+        <string id="V6RyPIQg" text="Schild zerschmettern III" original="Shatter Shield III"/>
         <!-- Shrug Off I -->
-        <string id="oyKzNISD" text="Achselzucken" original="Shrug Off I"/>
+        <string id="oyKzNISD" text="Achselzucken I" original="Shrug Off I"/>
```

Comment:
I didn't find a translation for that which makes sense. Don't think that anyone understands what `Achselzucken` does.

Reply:
This power name is tricky, it is a chance of immunity to cancel your action when you are hit by an attack. You "shrug off" the attack like it was nothing and go on with your action.

Reply:
> to cancel your action

Do you mean the attacker's action?
Like you got attacked and there is for example a 10 % chance that you don't get any damage (because the attacker's action was cancelled)?

Reply:
No, you still get damage. 
This is specific to ignoring the cancel action of an attack against your hero. 

Like when you swing for an attack and get hit by an arrow your swing is cancelled and you need to swing again from the start, with this skill you can ignore the cancel action.